types/react-simple-maps/index.d.ts

```diff
@@ -90,7 +90,7 @@ interface GeographiesChildrenArgument {
     projection: GeoProjection;
 }
 
-export interface GeographiesProps extends React.SVGAttributes<SVGGElement> {
+export interface GeographiesProps extends Omit<React.SVGAttributes<SVGGElement>, 'children'> {
```

Comment:
Previously we would intersect the renderprops with `ReactNode`. But there isn't actually a valid intersection unless `ReactNode` includes `{}`. Docs only mention renderprops as well: 

> children | Function

-- https://www.react-simple-maps.io/docs/geographies/